modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -244,4 +242,4 @@ Configuration properties can be overwritten. The Mule runtime engine uses the fo
 
 So, for example, if a configuration property named `size` is defined as a system property, and there is also an application configuration property named `size`, the value for the application is the value of the property with the most precedence (in this case, the system property).
 
-Also, a property can derive its value from other properties that are higher in the hierarchy. Therefore, an application property can derive its value from environment, system, and/or deployment properties. A system property can derive its value from a deployment property, and so on. For example, if there is a system property named `env`, an application configuration property can have the value `file.$&#123;env&#125;.xml`.  On the other hand, an application property cannot depend on an application property's value unless it's defined in the same file. For example, a Secure Configuration Property cannot depend on a Configuration Property. 
+Also, a property can derive its value from other properties that are higher in the hierarchy. Therefore, an application property can derive its value from environment, system, and/or deployment properties. A system property can derive its value from a deployment property, and so on. For example, if there is a system property named `env`, an application configuration property can have the value `file.$&#123;env&#125;.xml`.  On the other hand, an application property cannot depend on an application property's value unless it's defined in the same file. For example, a Secure Configuration Property cannot depend on a Configuration Property.
```

Comment:
We are not supposed to use and/or. In this case, I think "and" fits better than "or".

"Property" should probably not be initial-capped in "Secure Configuration Property" and "Configuration Property." I would look at the doc to see if there is any standard for this.